dpkgfiles/make_deb_package.README

```diff
@@ -0,0 +1,48 @@
+Instructions for using make_deb_package.py outside of Gitian
+
+
+1) We use cowbuilder to build in a chroot, so you need to prepare a chroot with
+   cowbuilder first (instructions from the Debian reproducible build project):
+
+      sudo cowbuilder --create --distribution codename --basepath /var/cache/pbuilder/base-reproducible.cow
+      sudo cowbuilder --login --save-after-exec --basepath /var/cache/pbuilder/base-reproducible.cow
+      echo 'deb http://reproducible.alioth.debian.org/debian/ ./' > /etc/apt/sources.list.d/reproducible.list
```

Comment:
It might be worthwhile to point out that the previous two lines are executed under the host OS and the remaining lines executed under the target. For that matter, it might help to explain at the beginning of the doc that the codename is the chroot/target being used.